legal-api/src/legal_api/services/colin.py

```diff
@@ -0,0 +1,37 @@
+# Copyright © 2020 Province of British Columbia
+#
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""This provides the service for colin-api calls."""
+import requests
+from flask import current_app
+
+from legal_api.services.bootstrap import AccountService
+
+
+class ColinService():
+    """Provides services to use the colin-api."""
+
+    @staticmethod
+    def query_business(identifier: str):
+        """Return a JSON object with business information."""
+        # Perform proxy call with identifier
+        url = f'{current_app.config["COLIN_URL"]}/businesses/{identifier}/public'
```

Comment:
```suggestion
        url = f'{current_app.config["COLIN_URL"]}/api/v1/businesses/{identifier}/public'
```
![image](https://github.com/user-attachments/assets/d0e45844-a18f-49b3-80ee-adaaeecc6ff8)